apps/sv/frontend/src/__tests__/sv.test.tsx

```diff
@@ -53,6 +53,66 @@ describe('SV user can', () => {
 
     expect(await screen.findByText('Vote Requests')).toBeDefined();
   });
+
+  test('see proper time format in popup', { timeout: 10000 }, async () => {
+    const mockedDate = new Date(2020, 0, 14, 4, 42, 0);
+
+    vi.setSystemTime(mockedDate);
+
+    const user = userEvent.setup();
+    render(<AppWithConfig />);
+
+    expect(await screen.findByText('Governance')).toBeDefined();
+    await user.click(screen.getByText('Governance'));
+
+    const dateInput = screen.getByTestId('datetime-picker-vote-request-expiration');
+
+    // Calendar button is flaky, at first it has today date and then after a slight delay it sets it up for the next week
```

Comment:
Just FYI this is intentional behaviour. The dates have expected defaults that come from the ledger but until that happens or if fetching those values fail, we show a default static date.